app/root.tsx

```diff
@@ -37,7 +37,6 @@ export const meta: V2_MetaFunction = () => [
 export const links: LinksFunction = () => [
   { rel: "stylesheet", href: tailwindStyles },
   { rel: "stylesheet", href: beehiveLoadingStyles },
-  // { rel: "stylesheet", href: scrollbarStyles }, // Turns out that scrollbar-gutter both-sides is buggy in Chrome :/
```

Comment:
I'd commented this rather than removing it because I was annoyed, but no reason to keep around unused code, it's safe in git. This reference and the file itself have both been deleted.